tasks/queue-granules/index.js

```diff
@@ -106,8 +109,8 @@ async function queueGranules(event, testMocks = {}) {
         granuleBatch,
         (queuedGranule) => {
           const collectionId = constructCollectionId(
```

Comment:
I imagine this actually needs to be valid for the call to updateGranule otherwise we'll write a strange collectionId to the granules.